checkov/terraform/checks/resource/aws/EKSPlatformVersion.py

```diff
@@ -25,7 +25,7 @@ def get_inspected_key(self) -> str:
 
     def get_expected_values(self) -> list[Any]:
         # https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
-        return ["1.24", "1.25", "1.26", "1.27", "1.28", "1.29", "1.30", "1.31"]
+        return ["1.24", "1.25", "1.26", "1.27", "1.28", "1.29", "1.30", "1.31", "1.32"]
```

Comment:
```suggestion
        return ["1.25", "1.26", "1.27", "1.28", "1.29", "1.30", "1.31", "1.32"]
```
May as well remove 1.24 since it's EOL tomorrow